testground/benchmark/benchmark/sendtx.py

```diff
@@ -1,68 +1,81 @@
+import json
 import time
 from concurrent.futures import ThreadPoolExecutor, as_completed
 
 import web3
 from eth_account import Account
+from hexbytes import HexBytes
 
-from .utils import gen_account, send_transaction
+from .utils import gen_account
 
 GAS_PRICE = 1000000000
+TMP_TXS_FILE = "tmp_txs.json"
 
 
-def sendtx(w3: web3.Web3, acct: Account, tx_amount: int):
-    initial_nonce = w3.eth.get_transaction_count(acct.address)
-    print(
-        "test begin, address:",
-        acct.address,
-        "balance:",
-        w3.eth.get_balance(acct.address),
-        "nonce:",
-        initial_nonce,
-    )
+def sendtx(w3: web3.Web3, tx):
+    txhash = w3.eth.send_raw_transaction(HexBytes(tx))
```

Comment:
will you generate and send cosmos-sdk raw tx directly? that'd reduce a little bit more overhead.

Reply:
I tried tx evm [raw](https://github.com/mmsqe/cronos/commit/8e116e32c39c6b55bc7d042beef358e65cff1f2f#diff-ba9ee2b4c34c9090bc6ab3331a9a09a06db4354f10bba5e5eb614045052f3cf0L108) was too slow and heavy